Source/monster.cpp

```diff
@@ -198,15 +198,15 @@ void GetLevelMTypes()
 		if (QuestStatus(QTYPE_BUTCH))
 			AddMonsterType(MT_CLEAVER, 2);
 		if (QuestStatus(QTYPE_GARB))
-			AddMonsterType(UniqMonst[0].mtype, 4);
+			AddMonsterType(UniqMonst[UMT_GARBUD].mtype, 4);

```

Comment:
```suggestion
			AddMonsterType(UniqMonst[UMT_GHARBAD].mtype, 4);
```

Reply:
Misspelled names were suggested by @AJenbo to be more true to original :)

Reply:
LOL

Reply:
Well at the very least we have to be consistent and rename them in `_mai_id` also in this case.

Reply:
I know, I just stopped adding new suggestions when you said it was intentional

Reply:
Abbreviations wheren't enough for them, they also needed to miss spell things :D